Makefile

```diff
@@ -61,9 +58,30 @@ help: ## Show this help screen
 		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
 	@echo ''
 
-function_list: ${BINARY} ## List all functions in generated binary file
+function-list: ${BINARY} ## List all functions in generated binary file
 	go tool objdump ${BINARY} | grep ^TEXT | sed "s/^TEXT\s//g"
 
-install_addlicense:
+install-addlicense:
 	[[ `command -v addlicense` ]] || go install github.com/google/addlicense
 
+golangci-lint: install-golangci-lint
+	golangci-lint run
+	glangci-lint fmt
+
+
+golangci-lint-fix: install-golangci-lint
+	@echo "Running linters and formatters with auto-fix...";
+	@echo "-----------------------------------------------------------------------"; 
+	@echo -e "\033[1;33mReview golangci-lint fixes and resolve any issues it couldn’t auto-fix\033[0m"
+	@echo "-----------------------------------------------------------------------"; 
+	golangci-lint run --fix
```

Comment:
I believe, `golangci-lint run` already runs the `fmt` command, but you will have to double check